octobot_trading/constants.py

```diff
@@ -110,6 +110,13 @@
                               "hitbtc", "kraken", "poloniex", "bitso", "ndax", "upbit",
                               "wavesexchange"]
 
+# from https://www.coingecko.com/en/categories/stablecoins
+USD_LIKE_COINS = [
+    "USDT", "USDC", "TUSD", "BUSD", "DAI", "USD",
+    "FRAX", "USDD", "USDP", "GUSD", "LUSD", "FDUSD",
+    "ALUSD", "USDTC", "MIM", "DOLA", "BOB", "MIMATIC"
+]
```

Comment:
I think we should add it into constants to make it available for every other modules

Reply:
It is already in constants or you mean something else ?

Reply:
Sorry I mean in commons

Reply:
good idea, i'll move it there